src/fcntl.rs

```diff
@@ -239,16 +273,30 @@ pub fn renameat<P1: ?Sized + NixPath, P2: ?Sized + NixPath>(
 #[cfg(all(target_os = "linux", target_env = "gnu"))]
 #[cfg(feature = "fs")]
 libc_bitflags! {
+    /// Flags for use with [`renameat2`].
     #[cfg_attr(docsrs, doc(cfg(feature = "fs")))]
     pub struct RenameFlags: u32 {
+        /// Atomically exchange `old_path` and `new_path`.
         RENAME_EXCHANGE;
+        /// Don't overwrite `new_path` of the rename.  Return an error if `new_path` already
+        /// exists.
         RENAME_NOREPLACE;
+        /// creates a "whiteout" object at the source of the rename at the same time as performing
+        /// the rename.
+        ///
+        /// This operation makes sense only for overlay/union filesystem implementations.
         RENAME_WHITEOUT;
     }
 }
 
 feature! {
 #![feature = "fs"]
+/// Like [`renameat`], but with an additional `flags` argument.
+///
+/// A `renameat2` call with a zero flags argument is equivalent to `renameat`.
```

Comment:
Maybe the following one is more suitable for our interface:
```suggestion
/// A `renameat2` call with an empty flags argument is equivalent to `renameat`.
```